### PR TITLE
publication statement bnf

### PR DESCRIPTION
**Pull request by rerowep**
Try to import a BNF record

### CODE REVIEW

File: rero_ils/modules/documents/dojson/contrib/unimarctojson/model.py [hunk 1]
```diff
@@ -17,29 +17,18 @@
...
+from dojson import utils
+from dojson.utils import GroupableOrderedDict, force_list
 from pkg_resources import resource_string
 
-unimarctojson = Overdo()
+from rero_ils.dojson.utils import ReroIlsOverdo, remove_trailing_punctuation
 
+unimarctojson = ReroIlsOverdo()
 
-# @unimarctojson.over('__order__', '__order__')
```
Comment (jma) [inline rero_ils/modules/documents/dojson/contrib/unimarctojson/model.py]: you can remove this comments

File: rero_ils/modules/documents/dojson/contrib/unimarctojson/model.py [hunk 2]
```diff
@@ -180,50 +169,106 @@ def unimarctoauthor(self, key, value):
...
+        publication = {
+            'type': type_per_ind2[ind2],
+            'statement': [],
+        }
+        subfields_d = utils.force_list(value.get('d'))
+        if subfields_d:
+            subfield_d = subfields_d[0]
+            publication['date'] = subfield_d
+            dates = subfield_d.replace('[', '').replace(']', '').split('-')
+            if dates[0]:
```
Comment (jma) [inline rero_ils/modules/documents/dojson/contrib/unimarctojson/model.py]: multiple if can probably be replaced by a `and` operator

Comment (rerowep) [reply rero_ils/modules/documents/dojson/contrib/unimarctojson/model.py]: replaced with `try` `except`

File: rero_ils/dojson/utils.py [hunk 3]
```diff
@@ -17,51 +17,51 @@
...
+    in order to be removed.
+    """
+    return re.sub(
+        r'([{0}]|\s+[{1}])$'.format(punctuation, spaced_punctuation),
+        '',
+        data.rstrip()).rstrip()
+
+
+class ReroIlsOverdo(Overdo):
+    """Specialzed Overdo.
```
Comment (BadrAly) [inline rero_ils/dojson/utils.py]: replace `Specialzed` by `Specialized`

File: rero_ils/dojson/utils.py [hunk 4]
```diff
@@ -114,6 +114,51 @@ def get_subfields(self, field, code=None):
             raise ValueError('data field expected (tag >= 01x)')
         return subfields
 
+
+class ReroIlsMarc21Overdo(ReroIlsOverdo):
+    """Specialzed Overdo.
```
Comment (BadrAly) [inline rero_ils/dojson/utils.py]: replace `Specialzed` by `Specialized`

File: rero_ils/dojson/utils.py [hunk 5]
```diff
@@ -114,6 +114,51 @@ def get_subfields(self, field, code=None):
...
+
+    This class adds RERO Marc21 proboerties and functions to the ReroIlsOverdo.
```
Comment (BadrAly) [inline rero_ils/dojson/utils.py]: replace `proboerties` by `properties`

**Pull request merged**